app/controllers/email_subscriptions_controller.rb

```diff
@@ -0,0 +1,60 @@
+class EmailSubscriptionsController < ApplicationController
+  include AuthenticatedApiConcern
+
+  before_action :check_subscription_exists, only: %i[show destroy]
+
+  def show
+    if email_subscription.email_alert_api_subscription_id
```

Comment:
Do we want any kind of error response if we can't find a subscription by the `subscription_name`?
I guess if the header is invalid they'll be logged out.

Is the problem detail response of any use here?

Reply:
There's the `before_action` which will return a 404